avatar/pandora_server.py

```diff
@@ -108,7 +108,13 @@ class AndroidPandoraServer(PandoraServer[AndroidDevice]):
     MOBLY_CONTROLLER_MODULE = android_device
 
     _instrumentation: Optional[threading.Thread] = None
-    _port: int = ANDROID_SERVER_GRPC_PORT
+    _port: int
+
+    def __init__(self, device: AndroidDevice) -> None:
+        global android_server_grpc_port
```

Comment:
Can we use `portpicker.pick_unused_port()` from the `portpicker` module (already in AOSP and used by mobly) instead ?